runtime/aardvarks/info.yaml

```diff
@@ -0,0 +1,79 @@
+apiVersion: apigeeregistry/v1
+items:
+  - apiVersion: apigeeregistry/v1
+    kind: API
+    metadata:
+      name: apigee-apihub-demo-fauna-aardvarks-proxy
+      labels:
+        apihub-business-unit: apigee-apihub-demo
+        apihub-kind: proxy
```

Comment:
So... "apihub-kind" hasn't been discussed much... are we saying that signal kinds are well-known across all signal sources? And that all signal kinds for all signal sources be registered in the apihub-kind taxonomy? As the source owns the signal, is there a reason to elevate this to apihub- level, or can we let this also be owned by the source?

Reply:
Yes, I think we should define a standard "kind" taxonomy that includes the set of signals that we expect to see. This is basically an accommodation to the existing taxonomy-based filtering in API Hub.

Reply:
I haven't been thinking about who's responsible for what -- but for this to work, the taxonomies have to match the values that are used, so it seems there should be a standard set that signal source can expect to exist. Maybe this is something to registry-check?

Reply:
Right. That's why I'm asking... if we're forcing signal kinds into a taxonomy, someone has to maintain those values so they match what signal sources can generate. (And, yes, registry-check does this checking.)